src/style/app.module.css

```diff
@@ -341,6 +357,7 @@
   background-color: transparent !important;
   border: 1px solid var(--brown-color);
   color: var(--brown-color) !important;
+  border-color: var(--brown-color) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with future attempts to having Dark Mode.
- For example `--search-button-bg` is much better than `--light-blue`